omod/src/main/java/org/openmrs/web/controller/user/UserFormController.java

```diff
@@ -142,7 +151,8 @@ public String handleSubmission(WebRequest request, HttpSession httpSession, Mode
 	        @RequestParam(required = false, value = "confirm") String confirm,
 	        @RequestParam(required = false, value = "forcePassword") Boolean forcePassword,
 	        @RequestParam(required = false, value = "roleStrings") String[] roles,
-	        @RequestParam(required = false, value = "createNewPerson") String createNewPerson,
+	        @RequestParam(required = false, value = "createNewPerson") String createNewPerson, 
+	        @RequestParam(required = false, value = "providerCheckBox") String addToProviderTableOption,
```

Comment:
@wluyima So do I let it be or change it? You're saying that if I change it to `Boolean`, it is automatically converted?